dbt_project.yml

```diff
@@ -21,6 +22,17 @@ vars:
     order: "{{ ref('stg_salesforce__order') }}"
     opportunity_line_item: "{{ ref('stg_salesforce__opportunity_line_item') }}"
 
+    account_history: "{{ ref('stg_salesforce__account_history') }}"
+    contact_history: "{{ ref('stg_salesforce__contact_history') }}"
+    event_history: "{{ ref('stg_salesforce__event_history') }}"
+    lead_history: "{{ ref('stg_salesforce__lead_history') }}"
+    opportunity_history: "{{ ref('stg_salesforce__opportunity_history') }}"
+    opportunity_line_item_history: "{{ ref('stg_salesforce__opportunity_line_item_history') }}"
```

Comment:
Same comment about removing history variables that are not used in the end

Reply:
Updated